packages/core/admin/admin/src/pages/Admin/Onboarding/index.js

```diff
@@ -195,6 +197,26 @@ const Onboarding = () => {
                     </TextLink>
                   </Stack>
                 ))}
+                <Stack horizontal spacing={3}>
+                  <Icon as={Message} color="primary600" />
+                  <TextLink
+                    as="a"
+                    href={
+                      communityEdition
```

Comment:
I wasn't able to find a way to include get help link in `DOCUMENTATION_LINKS` array for `Onboarding` & in `socialLinks` array for `Homepage` since I need access to `useAppInfos` hook to determine which href to render
do you see another solution for it?
(cc @gu-stav @joshuaellis)

Reply:
Maybe im not understanding the whole context but...

```ts
const docLinks = [...DOCUMENTATION_LINKS, { label: {}, icon: Message, href: communityEdition ? "" : "" }]
```

would work?

Reply:
I'd like it better that way indeed, thanks!